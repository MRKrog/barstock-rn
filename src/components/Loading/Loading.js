import React, { Component } from 'react';
import { Button, Image, View, Text, StyleSheet } from 'react-native';

class Loading extends Component {
  render() {

    return (
      <View style={styles.container}>
        <Image source={require('../../../assets/loading.gif')} />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    backgroundColor: '#ffffff',
    flex: 1,
    justifyContent: "center",
    alignItems: "center"
  },

  loadingText: {
    color: 'white',
    fontSize: 40,
    fontWeight: 'bold'
  },
})

export default Loading
