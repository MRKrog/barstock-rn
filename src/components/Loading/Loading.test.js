import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading.js';

describe('Button', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow(<Loading />)
            expect(component).toMatchSnapshot()
        });
    });
});
