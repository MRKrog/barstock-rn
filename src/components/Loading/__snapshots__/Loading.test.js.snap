// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Rendering should match to snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../assets/loading.gif",
      }
    }
  />
</View>
`;
